src/predict/predict_stacked.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
```

Comment:
stacked vs unstacked should be one file with an argument but much easier to do separately for now - maybe we can change this in future! leaving like this for now